frontend/src/components/admin/ApplicationProperties.js

```diff
@@ -0,0 +1,119 @@
+import React, { useContext, useEffect, useState } from "react";
+import {
+  Grid,
+  Heading,
+  Column,
+  Section,
+  TextInput,
+  Button,
+  Loading,
+} from "@carbon/react";
+import { FormattedMessage, useIntl } from "react-intl";
+import {
+  getFromOpenElisServer,
+  postToOpenElisServerJsonResponse,
+} from "../utils/Utils";
+import PageBreadCrumb from "../common/PageBreadCrumb";
+import { NotificationContext } from "../layout/Layout";
+import { AlertDialog } from "../common/CustomNotification";
+
+const ApplicationProperties = () => {
+  const intl = useIntl();
+  const { notificationVisible, addNotification, setNotificationVisible } =
+    useContext(NotificationContext);
+
+  const [loading, setLoading] = useState(true); // Set loading initially to true
+  const [applicationProperties, setApplicationProperties] = useState(null);
+  const [error, setError] = useState(null); // State to handle errors
+
+  useEffect(() => {
+    getFromOpenElisServer(
+      "/rest/properties",
+      (fetchedApplicationProperties, err) => {
+        if (err) {
+          setError(err);
+          setLoading(false); // Set loading to false in case of error
+        } else {
+          setApplicationProperties(fetchedApplicationProperties);
+          setLoading(false); // Set loading to false after data is fetched
+        }
+      },
+    );
+  }, []);
+
+  const handleSubmit = () => {
+    setLoading(true);
+    const url = `/rest/properties`;
+    let body = JSON.stringify({
+      "facilitylist.schedule.fixedRate": applicationProperties.fixedRate,
+    });
+    postToOpenElisServerJsonResponse(url, body, (response, err) => {
+      console.log("response from server", response);
+    });
+    addNotification({
+      kind: "success",
+      message: intl.formatMessage({ id: "message.propertiesupdate.success" }),
+      title: intl.formatMessage({ id: "notification.title" }),
+    });
+    setNotificationVisible(true);
+
+    setLoading(false);
+  };
+
+  return (
+    <>
+      {notificationVisible === true ? <AlertDialog /> : ""}
+      <PageBreadCrumb breadcrumbs={[{ label: "home.label", link: "/" }]} />
+      <Grid fullWidth={true}>
+        <Column lg={16}>
+          <Section>
+            <Section>
+              <Heading>
+                <FormattedMessage
+                  id="ewfggwgewgewgweg"
+                  defaultMessage="Application Properties"
+                />
+              </Heading>
+            </Section>
+          </Section>
+        </Column>
+      </Grid>
+      {loading && <Loading />}{" "}
+      {error && <p>Error: {error}</p>}{" "}
+
+      <div className="orderLegendBody">
+        <Grid>
+          <Column lg={10}>
+            <TextInput
+              value={applicationProperties?.fixedRate || ""}
+              labelText={intl.formatMessage({
+                id: "schedule.fixedRate",
```

Comment:
And also , you dont need to hard code the propertes in the UI , 
you can dymacaly fetch them from the backend